### PR TITLE
feat(bulk) : line SIP restrictions

### PR DESCRIPTION
**Pull request by JeremyDec**
## Title of the Pull Requests 
Bulk action Line SIP restrictions 

### Description of the Change
Adding Line SIP restrictions bulk action

### CODE REVIEW

File: client/components/telecom/telephony/bulkAction/telephony-bulk-action.less [hunk 1]
```diff
@@ -51,4 +51,14 @@
             }
         }
     }
+
+    .has-update {
+        background-color: #dfeaf8 !important;
```
Comment (antleblanc) [inline client/components/telecom/telephony/bulkAction/telephony-bulk-action.less]: `#dfeaf8` => `@oui-color-info-fade`

File: client/app/telecom/telephony/line/management/restrictions/telecom-telephony-line-management-restrictions.controller.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-angular.module("managerApp").controller("TelecomTelephonyLineRestrictionsCtrl", function ($stateParams, $timeout, $q, $document, OvhApiTelephony, ToastError, IpAddress, OvhApiMe) {
+angular.module("managerApp").controller("TelecomTelephonyLineRestrictionsCtrl", function ($stateParams, $timeout, $q, $document, $translate, OvhApiTelephony, OvhApiTelephonyLineOptions, ToastError, IpAddress, OvhApiMe, Toast, telephonyBulk) {
```
Comment (Jisay) [inline client/app/telecom/telephony/line/management/restrictions/telecom-telephony-line-management-restrictions.controller.js]: ```OvhApiTelephonyLineOptions``` can be removed => not used

File: client/app/telecom/telephony/line/management/restrictions/telecom-telephony-line-management-restrictions.html [hunk 3]
```diff
@@ -70,6 +70,19 @@ <h1 data-translate="telephony_line_restrictions_title"></h1>
...
                         </div>
+
+                        <telephony-bulk-action data-billing-account="{{ LineRestrictionsCtrl.bulkDatas.billingAccount }}"
+                                           data-service-name="{{ LineRestrictionsCtrl.bulkDatas.serviceName }}"
+                                           data-custom-class="link d-block pl-0 mb-3"
+                                           data-ng-disabled="lineOptionsForm.$invalid"
+                                           data-bulk-infos="LineRestrictionsCtrl.bulkDatas.infos"
+                                           data-filter-services="LineRestrictionsCtrl.filterServices"
+                                           data-get-bulk-params="LineRestrictionsCtrl.getBulkParams"
+                                           data-on-open="LineRestrictionsCtrl.applyLineChanges"
```
Comment (Jisay) [inline client/app/telecom/telephony/line/management/restrictions/telecom-telephony-line-management-restrictions.html]: Why do you need to apply changes to current service on bulk modal opening ?

File: client/app/telecom/telephony/line/management/restrictions/telecom-telephony-line-management-restrictions.controller.js (hunk 2 above)
Comment (JeremyDec) [reply client/app/telecom/telephony/line/management/restrictions/telecom-telephony-line-management-restrictions.controller.js]: done

**Pull request merged**